snappy_wrappers/wrappers/gatk4_hc/environment.yaml

```diff
@@ -2,9 +2,9 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - gatk4
-  - bcftools =1.16
-  - samtools =1.16
-  - htslib =1.16
-  - parallel =20221122
-  - picard-slim =2.27
+  - gatk4==4.5.0.0
+  - bcftools==1.16
+  - samtools==1.16.1
+  - htslib==1.16
+  - parallel==20221122
```

Comment:
I have pinned version 20231122 in a local branch, I think that might better here